media-video/syncplay/syncplay-1.6.4a.ebuild

```diff
@@ -18,32 +18,30 @@ IUSE="+client +server vlc"
 REQUIRED_USE="vlc? ( client )
 	${PYTHON_REQUIRED_USE}"
 
-DEPEND=""
 # TODO: investigate the possibility of enabling PyQt5 gui
 # possible licensing concerns
 RDEPEND="${PYTHON_DEPS}
-	>=dev-python/certifi-2018.11.29
+	>=dev-python/certifi-2018.11.29[${PYTHON_USEDEP}]
 	>=dev-python/twisted-16.4.0[crypt,${PYTHON_USEDEP}]
-	>=dev-python/zope-interface-4.4.0
+	>=dev-python/zope-interface-4.4.0[${PYTHON_USEDEP}]
 	vlc? ( media-video/vlc[lua] )"
 
 src_prepare() {
 	default
-	sed -i 's/"noGui": False,/"noGui": True,/' \
-		syncplay/ui/ConfigurationGetter.py \
-		|| die "Failed to patch ConfigurationGetter.py"
+	sed -i '/"noGui": /s/False/True/' syncplay/ui/ConfigurationGetter.py ||
+		die "Failed to patch ConfigurationGetter.py"
 }
 
 src_compile() {
 	:
 }
 
 src_install() {
-	local MY_MAKEOPTS=( DESTDIR="${D}" PREFIX=/usr )
-	use client && \
-		emake "${MY_MAKEOPTS[@]}" VLC_SUPPORT=$(usex vlc true false) install-client
-	use server && \
-		emake "${MY_MAKEOPTS[@]}" install-server
+	local my_makeopts=( DESTDIR="${D}" PREFIX=/usr )
+	use client &&
```

Comment:
Minor suggestion: you could wrap this all in a single `emake` call, e.g.:

    emake ... $(usex client install-client '') $(usex server install-server '')

or by constructing an array of install targets beforehand.